nodl_python/nodl/exception.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2020 Canonical Ltd.
```

Comment:
I'm going to flip-flop. I think this file should be `errors.py` instead.

Reply:
Fixed in db8e13c